services/src/datasets/external/nature40.rs

```diff
@@ -464,6 +465,7 @@ impl MetaDataProvider<GdalLoadingInfo, RasterResultDescriptor, RasterQueryRectan
                 Some(self.auth().to_vec()),
             )?,
             result_descriptor: raster_descriptor_from_dataset(&dataset, band_index as isize)?,
+            cache_ttl: CacheTtl::Seconds(15 * 60),
```

Comment:
constant

Reply:
I changed this to using the ttl from the provider definition 5c04d280f009ec129d8b8fb5a92e3fc1ca61c65b